src/projectscene/view/trackspanel/trackslistmodel.cpp

```diff
@@ -92,6 +92,8 @@ void TracksListModel::load()
         m_trackList.push_back(buildTrackItem(track));
     }
 
+    onSelectedTracks(selectionController()->selectedTracks());
```

Comment:
Why receiving this event is not enough?
```
selectionController()->tracksSelected().onReceive(this, [this](trackedit::TrackIdList tracksIds) {
        onSelectedTracks(tracksIds);
    });
 ```

Reply:
because undo/redo reloads whole model and does it asynchronously
```c++
muse::async::Async::call(this, [this]() {
        load();
    });
``` 

so the order looks like this:
1. call reload()
2. setTracksSelected() -> emit signal that tracks changed
3. signal is handled where it's connected
4. reload is done here (it's too late to catch and handle selectedChanged signal)

so I set selectedTracks manually here